### ISSUE TITLE
Correlations lesson should be explicit about "don't just trust your eyeballs"

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/curriculum#1399]
- [x] Add to notes for facilitator
- [x] Add to lesson section overviews
- [ ] Maybe introduce `r-squared` contract?

**Comment by flannery-denny** [comment]
@schanzer

1. I've pushed suggested wording changes for this lesson. Please review.
2. I changed the directions in the Desmos activity to read "Sort the cards into 3 piles by the apparent strength of the correlations."
3.  I'm noticing that this sentence has a lot of bolded words that are not in our glossary, should we add any of them?

> We can analyze a single quantitative variable, such as age or pounds to identify a value that is **typical**, how much the values **vary**, and what kind of values are **usual** or **unusual**.

4. I would suggest that we switch the order of **Correlations in the Animals Dataset** and **Correlation. This is a screenshot of the section of the lesson in question. 
<img width="918" alt="Screen Shot 2023-06-23 at 1 00 24 PM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/bfce08af-b4af-4a43-8fb4-0b04bb790794">
does not imply causation**. 
5. I'm unclear on what you envision with the other bullets. Close if you're satisfied.

**Comment by schanzer** [comment]
@flannery-denny 

1. wording changes look good
2. directions changes look good
3. They should not be in the glossary. They are bolded as a hint to the facilitator, but I've added some explanation in my recent commit.
4. I like this change! Go for it
5. Added notes for the facilitator to the top, and a "common misconceptions" section to the relevant activity. Let's ignore R-squared.

If you're happy with my additions, then close after you've switched the order in point 4.
